package.json

```diff
@@ -255,37 +255,49 @@
       "default": "automatic",
       "order": 17
     },
+    "openerSplit": {
```

Comment:
You know me, I like explicit and verbose names; my vote is for `openerSplitDirection`. Can't really think of a better alternative.